src/pac_scan.ll

```diff
@@ -223,6 +223,11 @@ ESCSEQ	(\\([^\n]|[0-7]{3}|x[[:xdigit:]]{2}))
 				return TOK_ID;
 				}
 
+<INITIAL,PP>{ID}::{ID}		{
```

Comment:
```suggestion
<INITIAL,PP>{ID}(::{ID})*		{
```

More generalized; could also then remove the above `{ID}` rule as covered by this.

Reply:
Fixed.